### DISCUSSION TITLE
displaying tif (fax) images with different horizontal and vertical DPI

### DISCUSSION THREAD

**Discussion by agleung** [sumatrapdfreader/sumatrapdf#2857] [Ideas]
Very excited to find a light weight document viewer for Windows. My use case is to view pdf and tif (faxes). It is not uncommon to have fax images with [horizontal DPI double that of vertical DPI](https://stackoverflow.com/questions/7348108/tiff-wrong-height). When that happens, the image rendered in sumatraPDF will look squashed. Would it be possible to implement a way to read the horizontal and vertical DPI and expand the vertical image height according? Thank you.

**Comment by kjk** [comment]
Can you provide a sample file for test?

**Reply by GitHubRulesOK** [reply]
@kjk there are different formats involved but here is most basic where fax printouts are for high speed or slow speed printing

[fax.zip](https://github.com/sumatrapdfreader/sumatrapdf/files/9642224/fax.zip) However note that both show same resolution

![image](https://user-images.githubusercontent.com/12754913/192176331-05235648-22fc-4113-91ee-b79ab6325c52.png)
![image](https://user-images.githubusercontent.com/12754913/192176494-906d614f-2496-4bc1-9db0-380f0db8a597.png)
